Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM python:2.7
-RUN apt-get update && apt-get install -y libjpeg62-turbo-dev postgresql-client nodejs npm
+RUN apt-get update && apt-get install -y libjpeg62-turbo-dev postgresql-client nodejs npm vim
 
 WORKDIR /pontoon
 
-RUN npm install -g yuglify
+RUN npm install -g yuglify babel
```

Comment:
I don't think `-g` worked with a package.json file when I tested it. I could be wrong though.